guide/docs/faq/general.mdx

```diff
@@ -0,0 +1,272 @@
+# General
+
+:::note
+
+This is placeholder content that has been taken directly from the [documentation](https://docs.disnake.dev/en/latest/), and inherited from `discord.py`. This section will most likely be rewritten in the future.
+
+:::
+
+General questions regarding library usage belong here.
```

Comment:
This should be above the note, just to follow the same syntax in all files

Reply:
Yep, agree with this